resources/begrep/sikkerDigitalPost/nyinf/api/openapi_spec.yaml

```diff
@@ -1,173 +1,173 @@
           content:
             multipart/form-data:
               schema:
                 type: object
                 properties:
-                  forettningsmelding: 
+                  forretningsmelding: 
                     type: string
-                    format: base64
-                    description: 'Digitalpost melding formatert som base64 encoded JWT'
+                    description: JWT med forretningsmelding i payload. Se https://todo.no
                   dokumentpakke: 
                     type: string
                     format: binary
-                    description: Kryptert dokumentpakke. Formate på dokumentpakke følger ASiC                    
+                    description: Kryptert dokumentpakke. Se https://todo.no
         responses: 
           201: 
-            description: Mottatt OK av aksesspunktleverandør, og vil bli sendt videre til postkasse
+            description: Foresendelse sendt til avsenders aksesspunkt.
           400: 
-            description: Valideringsfeil - utdypende informasjon om feilen beskrevet i problem+json
+            description: Valideringsfeil. Utdypende informasjon om feilen beskrevet i problem+json.
             content:
               application/problem+json:
                 schema: 
                   $ref : '#/components/schemas/problem'
           401:
-            description: Utløpt token
+            description: Autentiseringsfeil. F.eks. manglande JWT bearer grant.
             content:
               application/problem+json:
                 schema: 
                   $ref : '#/components/schemas/problem'
           403:
-            description: Token mangler "digitalpostinnbygger:send" scope, eller avsender har ikke gyldig avtale med aksesspunktleverandør.
+            description: Autorisasjonsfeil. F.eks. gyldig JWT bearer grant som mangler påkrevd scope.
             content:
               application/problem+json:
                 schema: 
                   $ref : '#/components/schemas/problem'   
     /{version}/statuses/{id}:
       get:
-        summary: Henter liste med statuser gitt foreendelsesid
+        summary: Hent statuser for foresendelse
         parameters: 
             - in: path
               name: version            
               schema:
                 type: string                               
-                description: Versjon av API
+                description: Versjon av APIet
               required: true
             - in: path
               name: id            
               schema: 
                 type: string
                 format: uuid
-                description: Unik ID til meldingen gitt i SBDH - documentIdentification - instanceIdentifier
+                description: Unik ID for forsendelsen. Gitt som del av standard business document header (SBDH) ved sending (<code>standardBusinessDocument.documentIdentification.instanceIdentifier</code>).
```

Comment:
Lurer litt på om det er ConversationIDen som skal brukes her? Ellers må vi sørge for at meldingsIDen som benyttes for sending er den samme som i IP. For øyeblikket genereres den. Hva synes du?

Reply:
Eller kanskje det beste er at IDen er lik den i IP. Jeg kan sende den inn,

Reply:
Eg har ikkje tenkt på det - veit ikkje bakgrunn for at det står slik det gjer, men synes gjort med ganske bevisst?

Kva er konsekvenser av at det er som beskrevet ?

Reply:
Me har konkludert med at det er sbdh.documentIdentification.instanceIdentifier som skal brukes, som beskrevet (og at det er samme documentIdentification.instanceIdentifier som sendes inn til integrasjonspunktet som blir sendt videre til aksesspunktet)